src/components/NcButton/NcButton.vue

```diff
@@ -607,7 +607,7 @@ export default defineComponent({
 		},
 
 		ncPopoverTriggerAttrs() {
-			return this.getNcPopoverTriggerAttrs()
+			return (this.getNcPopoverTriggerAttrs as () => Record<string, string|undefined>)()
```

Comment:
When using Options API it is always infered as `unknown` only Composition API (setup) will provide correct type information without any need to cast it here.